cfgov/unprocessed/js/molecules/Notification.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+require( '../modules/polyfill/class-list' );
+
+// Required modules.
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+
+/**
+ * Notification
+ * @class
+ *
+ * @classdesc Initializes a new Notification molecule.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the molecule.
+ * @returns {Object} An Notification instance.
+ */
+function Notification( element ) { // eslint-disable-line max-statements, inline-comments, max-len
+
+  var BASE_CLASS = 'm-notification';
+
+  // Bitwise flags for the state of this Expandable.
+  var SUCCESS = 'success';
+  var WARNING = 'warning';
+  var ERROR = 'error';
+
+  var _dom =
+    atomicCheckers.validateDomElement( element, BASE_CLASS, 'Notification' );
+  var _icon = _dom.querySelector( '.' + BASE_CLASS + '_icon' );
+  var _content = _dom.querySelector( '.' + BASE_CLASS + '_content' );
```

Comment:
If all three of these are dom elements, should we have a naming convention for denoting that like we did with jquery? like `_icon_dom`, `_content_dom`?